python/cuspatial/cuspatial/core/spatial/join.py

```diff
@@ -311,7 +313,8 @@ def quadtree_point_to_nearest_linestring(
         between the two.
 
         point_index : cudf.Series
-            Indices of each point that intersects with a linestring.
+            Indices of each point (in ``point_indices``) that intersects with
+            a linestring.
         linestring_index : cudf.Series
             Indices of each linestring with which a point intersected.
         distance : cudf.Series
```

Comment:
I'm providing the whole suggested return documentation here because it runs outside the range of lines I can comment on in this PR.  Please make analogous changes to the PiP one as mentioned. Also, please double check that the right double quotes are being used. I'm not that familiar with Python docstring syntax.

```
    result : cudf.DataFrame
        Indices for each point and its nearest linestring, and the distance
        between the two.

        point_index : cudf.Series
            Index of point. This index refers to ``point_indices``, so it is an index to an index.
        linestring_index : cudf.Series
            Index of the nearest linestring to the point.
        distance : cudf.Series
            Distance between point and its nearest linestring.
```